modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -145,9 +160,25 @@ export default {
       return (this.disabled || this.busy);
     }
   },
+  mounted() {
+    if (this.type === 'color') {
+      this.previewColor();
+    }
+  },
   methods: {
     click($event) {
       this.$emit('click', $event);
+    },
+    previewColor() {
```

Comment:
I think it would be more maintainable to actually chuck a span for this purpose in the component and generate its dynamic styles the more usual vue way, unless I'm missing something.

Reply:
:chefs-kiss-goodbye: